app/models/trip.rb

```diff
@@ -29,6 +28,9 @@ def end_station_longitude
   scope :longest_ride, -> {maximum(:duration)}
   scope :shortest_ride, -> {minimum(:duration)}
   scope :average_ride_length, -> {average(:duration).to_f.round(2)}
-
+  scope :most_popular_starting_station, -> {maximum(:start_station)}
+  scope :most_popular_ending_station, -> {maximum(:end_station)}
+  scope :date_with_most_rides, -> {group(:start_date).order('count(*) DESC').count.first.first}
+  scope :date_with_most_rides_trip_count, -> {group(:start_date).order('count(*) DESC').count.first.last}
```

Comment:
Definitely want to review the .first.first thing. Just seems like a red flag that there is a better way to do this.

Reply:
This isnt really how I want to use a scope, @Maxscores. ` The method is intended to return an ActiveRecord::Relation object, which is composable with other scopes` per the docs...so if i am returning a number, i am not properly using a scope. lets talk about it tomorrow

Reply:
@maxscores